src/main/sensors/gyro.c

```diff
@@ -76,6 +76,9 @@
 #ifdef USE_GYRO_DATA_ANALYSE
 #include "sensors/gyroanalyse.h"
 #endif
+#ifdef USE_RPM_FILTER
```

Comment:
Is there a compilation problem if this isn't conditional? Headers are normally included unconditionally in Betaflight. (`USE_GYRO_DATA_ANALYSE` above is a special case, as it includes `arm_math.h` that does not exist on non-ARM, and hence breaks the SITL build and the tests.)

Reply:
I don't remember. Let's try. I might have just erred conservatively on more conditionals this time around. 😄